src/checkers/inference/VariableAnnotator.java

```diff
@@ -1640,7 +1650,7 @@ private Slot getOrCreateDeclBound(AnnotatedDeclaredType type) {
             if (classTree != null) {
                 final AnnotatedDeclaredType declType = inferenceTypeFactory.fromElement(classElt);
                 declSlot = replaceOrCreateEquivalentVarAnno(declType, classTree, treeToLocation(classTree));
-                classDeclAnnos.put(classElt, (SourceVariableSlot) declSlot);
+                classDeclAnnos.put(classElt, declSlot);
```

Comment:
When is this not a `SourceVariableSlot`? As it's not explicitly annotated, shouldn't it always be a SVS?